Adds test workflows to github in markdown

taywrobel opened the pull request:
Adds a markdown version of the updated test
script and workflows to github, so that it
can be maintained by the entire team and reviewd
by beta participants.

Adds script to git - fixes #251
Removes incorrect disband check in test 35 - fixes #202
Includes check for toast when removing last song - fixes #246
Adds reconnect to test 24 - fixes #247

taywrobel committed:
Adds test workflows to github in markdown

Adds a markdown version of the updated test
script and workflows to github, so that it
can be maintained by the entire team and reviewd
by beta participants.

taywrobel committed:
Change test script to use normal quotes

It used those fancy quotes before, which
don't render with some fonts.  Looked fine
on github, but terrible in source control,
so they have been replaced with the ASCII
quotes.

taywrobel committed:
Change test script to use normal quotes

It used those fancy quotes before, which
don't render with some fonts.  Looked fine
on github, but terrible in source control,
so they have been replaced with the ASCII
quotes.

taywrobel committed:
Merge branch 'testScript' of github.com:twrobel3/SoundStream into testScript

taywrobel committed:
Change back to fancy quotes for test script

It looked better with them.

theJenix commented on docs/testWorkflows.md:
```diff
@@ -0,0 +1,686 @@
+- Press the �Disconnect� button on phone B
+- Press �Create� on phone B
+- Press the �Play� icon in the playbar on phone A
+- Reconnect phone B to phone A's Network
```
This should reference a test procedure (e.g. Complete Test 9) or be broken out into steps.

theJenix commented on docs/testWorkflows.md:
```diff
@@ -0,0 +1,686 @@
+- Press the �Play� icon in the playbar on phone A
+
+### Expected Results: ###
+ Upon removal of last song, a toast appears instructing the user to add songs to the playlist. Upon pressing play, there is no song in the playlist and a toast appears saying �Playlist is empty�
```
This needs to identify which phone the "Playlist is empty" toast appears on.

theJenix commented on docs/testWorkflows.md:
```diff
@@ -0,0 +1,686 @@
+- Press the �Play� icon in the playbar on phone B
+
+### Expected Results: ###
+ Upon removal of last song, a toast appears instructing the user to add songs to the playlist. Upon pressing play, there is no song in the playlist and a toast appears saying �Playlist is empty�
```
This needs to identify which phone the "Playlist is empty" toast appears on.

theJenix commented:
The tests that result in an empty playlist must indicate which phone (guest or host) will display the "Playlist is empty" message.

reidbaker commented:
I would love to get rid of all the question marks. That should not hold off getting this pulled in but if it doesnt get fixed please make a ticket.

dgreenhalgh commented:
@twrobel3,could we get the changes made and have this pulled in by this Sunday's meeting? I'd love to start looking forward at what's next.

taywrobel commented:
Yeah, it's been a crazy week, but I can take care of it tomorrow.

On Fri, Aug 9, 2013 at 8:45 AM, David Greenhalgh
notifications@github.comwrote:

> @twrobel3 https://github.com/twrobel3,could we get the changes made and
> have this pulled in by this Sunday's meeting? I'd love to start looking
> forward at what's next.
> 
> —
> Reply to this email directly or view it on GitHubhttps://github.com/TheLastCrusade/SoundStream/pull/259#issuecomment-22403100
> .

## 

Taylor Wrobel
- www.linkedin.com/in/taylorwrobel/

dgreenhalgh commented:
No worries, thanks man
On Aug 9, 2013 3:44 PM, "Taylor Wrobel" notifications@github.com wrote:

> Yeah, it's been a crazy week, but I can take care of it tomorrow.
> 
> On Fri, Aug 9, 2013 at 8:45 AM, David Greenhalgh
> notifications@github.comwrote:
> 
> > @twrobel3 https://github.com/twrobel3,could we get the changes made
> > and
> > have this pulled in by this Sunday's meeting? I'd love to start looking
> > forward at what's next.
> > 
> > —
> > Reply to this email directly or view it on GitHub<
> > https://github.com/TheLastCrusade/SoundStream/pull/259#issuecomment-22403100>
> > 
> > .
> 
> ## 
> 
> Taylor Wrobel
> - www.linkedin.com/in/taylorwrobel/
> 
> —
> Reply to this email directly or view it on GitHubhttps://github.com/TheLastCrusade/SoundStream/pull/259#issuecomment-22418879
> .

taywrobel committed:
Removed quotes and updated script for corectness

Removed quotation marks from around strings in tests.
They are descriptive enough of functionality without
them.  Also fixed expected results of tests that empty
the playlist to describe which phone the empty playlist
toast should appear on.

taywrobel commented:
@theJenix could you take a look and make sure that all the changes were addressed?  If so this should be good to push in.

theJenix commented:
:+1: Looks good to me. :pushpin:
